pkg/transfers/inbound/cleanup_test.go

```diff
@@ -43,3 +74,45 @@ func TestCleanupErr(t *testing.T) {
 		t.Errorf("unexpected deleted file: %s", agent.DeletedFile)
 	}
 }
+
```

Comment:
These tests don't seem to fully cover `DeleteZeroByteFiles`, so can we increase the coverage a bit? 

See: https://codecov.io/gh/moov-io/paygate/compare/78351c715af5bf93e2ba35a338ba0b3dffed11d0...a1fe7ba356d8162d5e48ffc7372aeef25596862c/diff